Plot/src/main/java/io/deephaven/plot/datasets/multiseries/AbstractMultiSeries.java

```diff
@@ -48,17 +48,22 @@
     private transient Set<Object> seriesKeys;
     private transient Map<String, Object> seriesNames;
 
-    private final transient java.util.function.Function<Object[], String> DEFAULT_NAMING_FUNCTION = key -> {
+    private final transient java.util.function.Function<Object, String> DEFAULT_NAMING_FUNCTION = key -> {
```

Comment:
There are a lot more changes to this than I would have expected, but maybe I'm failing to understand the nature of the issue.
I do see one error in the previous version, which is that I was using `Objects.toString(key)` instead of `Objects.toString(key[0])` in the unary case.
I considered including a `null` check, and rejected it because I didn't see an obvious code path to need it. If it's necessary, so be it.
I'm not sure why you've reverted the signature change.

Reply:
Resolved by cherry-picking your commit that made this change.